### PR TITLE
Update maintainer tools

### PR DESCRIPTION
**Pull request by jwhitlock**
Add some maintainer tools that have been useful on https://github.com/django-nose/django-nose and other projects.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+	coverage run ./runtests.py
+	coverage report -m
+
+.PHONY: coverage
+coverage: coverage-console
+	coverage html
+	open htmlcov/index.html
+
+.PHONY: release
+release: sdist
```
Comment (edmorley) [inline Makefile]: Perhaps wheels should be generated here too?

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+.PHONY: test-release
+# Add [test] section to ~/.pypirc, https://test.pypi.org/legacy/
+test-release: sdist
+	twine upload --repository test dist/*
+	python -m webbrowser -n https://testpypi.python.org/pypi/django-pylibmc
+
+.PHONY: sdist
+sdist: clean
+	python setup.py sdist bdist_wheel
```
Comment (jwhitlock) [inline Makefile]: Here is where the wheel is generated, and is a pre-condition for release.

Comment (edmorley) [reply Makefile]: Oh I took the `dist` makefile target too literally and didn't check what it ran! 😄

Comment (edmorley) [reply Makefile]: sdist, even

**Pull request merged**